### ISSUE TITLE
Display Sync causes stutter with Hardware Decoding and TrueHD

### ISSUE THREAD

**Issue by DaveBinM** [plexinc/plex-media-player#573]
## Environment
- **Platform(s)**: macOS (Late 2014 Mac Mini)
- **OS Version(s)**: 10.12.5
- **PMP Release**: 1.3.0 +

## Impact to users
1. Video and audio stutters and cuts in and out

## Steps to reproduce
1. Enable `Hardware Decoding` (modern or old)
2. Enable `Display Sync (resample audio)`
3. Play a movie with TrueHD (see sample file [here](https://www.amazon.com.au/clouddrive/share/u0HWDads5hd5EHlUswS5WpkSLESYLm5SPu2D4Xbgb5S?ref_=cd_ph_share_link_email))
4. Start playback

## Expected behaviour
1. PMP will play back the video smoothly

## Actual behaviour
1. The video stutters, with audio dropping out.

## Screenshots and logs
Logs from @atrus05, where he restarted PMP to get a clean log. Then fired up the sample. Issues started directly. After 5 seconds he enabled the lavfi dynaudnorm filter, and the video directly started behaving again.

[Plex Media Player.log.zip](https://github.com/plexinc/plex-media-player/files/1044624/Plex.Media.Player.log.zip)

![image uploaded from ios](https://cloud.githubusercontent.com/assets/19317607/26678635/c6234424-4714-11e7-8888-d14fc1686c5c.jpg)

Video showing behaviour [here](https://www.amazon.com.au/clouddrive/share/LCwknRwkrCKf4zhUqt1PMtOWxQTv5oXX6jJIH2aHt8X?ref_=cd_ph_share_link_copy).

**Comment by atrus05** [comment]
As Dave is mentioning: I was able to reproduce on my 2012 quad i7 Mac Mini with OSX 10.12.X

**Comment by DaveBinM** [comment]
Also been reported by a forum user here: http://forums.plex.tv/discussion/278684/pmp-1-3-4-mac-os-and-dolby-truehd-problems#latest

**Comment by LongChair** [comment]
@DaveBinM : is this stll an issue with recent builds ?

**Comment by DaveBinM** [comment]
@LongChair Unfortunately I don't have a Mac connected to a display with the appropriate refresh rates available to test this any more. Perhaps @atrus05 might be able to assist?

**Comment by LongChair** [comment]
@atrus05 : could you eventually doublecheck this one in recent builds ? :)

**Comment by atrus05** [comment]
@DaveBinM Can you Slack DM me which video we were trying with so I can try that sample again? I seem to remember me playing from your server.

**Comment by atrus05** [comment]
Heh, the sample file was linked in the original post. Need better glasses.

Unfortunately a lot has changed in my setup since I tried this.  New OS (10.13.X) and no receiver being the biggest two changes. And unfortunately one or more of the changes to my setup has caused that I no longer can reproduce this. Not even with the version we tried this with back a year ago (659). So I can’t be a good boy for this issue. Maybe lock this one for now until we have found some way to reproduce this?
